src/Confluent.Kafka/Timestamp.cs

```diff
@@ -27,6 +27,11 @@ namespace Confluent.Kafka
     public struct Timestamp
     {
         /// <summary>
+        ///     Unix epoch as UTC DateTime
+        /// </summary>
+        public static readonly DateTime UnixEpochUtcDateTime = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
UinxTimeEpoch or UnixTimestampEpoch? (we may want to be consistant with other methods)
Also wanted to specify it is Utc as it's a Datetime, and Utc vs Local vs Unspecified is misleading